masci_tools/io/parsers/banddos.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+###############################################################################
+# Copyright (c), Forschungszentrum Jülich GmbH, IAS-1/PGI-1, Germany.         #
+#                All rights reserved.                                         #
+# This file is part of the Masci-tools package.                               #
+# (Material science tools)                                                    #
+#                                                                             #
+# The code is hosted on GitHub at https://github.com/judftteam/masci-tools    #
+# For further information on the license, see the LICENSE.txt file            #
+# For further information please visit http://www.flapw.de or                 #
+#                                                                             #
+###############################################################################
+
+"""
+Parser for the FLEUR (MaX_R5) `banddos.hdf`  file
+"""
+
+import numpy as np
+import h5py
+
+
+def hdfList(obj, sep="\t"):
+    """
+    Iterate through groups in a HDF5 file and prints...
+    ...the groups and datasets names and datasets attributes
+    """
+    if type(obj) in [h5py._hl.group.Group, h5py._hl.files.File]:
+        for key in obj.keys():
+            print(sep, '-', key, ':', obj[key])
+            hdfList(obj[key], sep=sep+'\t')
+    elif type(obj) == h5py._hl.dataset.Dataset:
+        for key in obj.attrs.keys():
+            print(sep+'\t', '-', key, ':', obj.attrs[key])
+
+
+def h5dump(path, group='/'):
+    """
+    group: you can give a specific group, defaults to the root group
+    """
+    with h5py.File(path, 'r') as f:
+        hdfList(f[group])
+
+
+def kpath(kcoord, reciprocalCell):
+    """
+    Construct k-point path from the k point coordinate array 
+    """
+    nkpt = kcoord.shape[0]
+    kpts = np.zeros(nkpt)
+    kpts[0] = np.linalg.norm(reciprocalCell.dot(kcoord[0]))
+    for i in range(1, nkpt):
+        kpts[i] = kpts[i - 1] + \
+            np.linalg.norm(reciprocalCell.dot(
+                kcoord[i]) - reciprocalCell.dot(kcoord[i - 1]))
+    return kpts
+
+
+def writeBand(inpFile, outFile1, outFile2):
+    eV = 27.211386
```

Comment:
We now have a module `masci_tools.util.constants`, where we define all the constants that should be used in masci-tools and the aiida-plugins

Reply:
You can use `masci_tools.util.constants.HTR_TO_EV` instead

Reply:
A the moment there are only a few functions in the file, for inspecting the `banddos.hdf` and to write the band to a file.  If there is demand I can convert this into a generalised parser Class for parsing the `hdf`.